xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -132,6 +156,35 @@ public ConfigurationParser(String xml) throws IOException, SAXException, JAXBExc
     DocumentBuilder domBuilder = factory.newDocumentBuilder();
     domBuilder.setErrorHandler(new FatalErrorHandler());
     Element dom = domBuilder.parse(xml).getDocumentElement();
+
```

Comment:
Woah... I'm swallow YAGNI for not limiting replacement to just our namespaces... but failed replacement should throw in configuration parsing.